### PR TITLE
Node Package Manager version enforced         GsSoC'23

### PR DESCRIPTION
**Pull request by Nikhil1017**
as per #413

### CODE REVIEW

File: components/RightSideBar/Notes.tsx [hunk 1]
```diff
@@ -1,14 +1,14 @@
-import React from "react";
-import Image from "next/image";
-type Notes = {
+import React from 'react';
+import Image from 'next/image';
+type NotesCard = {
```
Comment (lalitmee) [inline components/RightSideBar/Notes.tsx:3]: Leave an empty line after imports

**Review by lalitmee — CHANGES_REQUESTED** [review]

File: components/RightSideBar/Notes.tsx (hunk 1 above)
Comment (Nikhil1017) [reply components/RightSideBar/Notes.tsx:4]: @lalitmee Thanks, I have left emty line now

**Comment by Nikhil1017** [comment]
@Vaishnavi2701-mk

**Review by lalitmee — APPROVED** [review]

**Comment by Nikhil1017** [comment]
@lalitmee conflicts resolved

File: components/depricated/RightSideBar/IndividualDate.tsx [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
   date: number;
   day: string;
   isTodayDate: boolean;
 }
 
-const IndividualDate = (props: IndividualDate) => {
+const IndividualDate = (props: IndividualDateCard) => {
   return (
     <div>
       <div
```
Comment (PiyushKalyanpy) [inline components/depricated/RightSideBar/IndividualDate.tsx:12]: Have you mentioned these changes in your issue #413  ?? If not please revert back the changes for all the files that were not mentioned in issue #413

File: components/depricated/RightSideBar/Notes.tsx
Comment (PiyushKalyanpy) [inline components/depricated/RightSideBar/Notes.tsx:1]: remove all the unnecessary files from the pr and keep only three files : 
1. .npmrc
2. .eslint.json
3. package.json

File: package-lock.json
Comment (PiyushKalyanpy) [inline package-lock.json:1]: create a new pr which should only contain the above mentioned files.

**Review by PiyushKalyanpy — CHANGES_REQUESTED** [review]

**Comment by Nikhil1017** [comment]
@PiyushKalyanpy I have modified the existing pull request to only include those 3 files. I have opened issue #450 addressing the changes to other files and pull request #451 solving the issue.

**Comment by Nikhil1017** [comment]
@PiyushKalyanpy

**Pull request closed without merging**